django/db/backends/mysql/features.py

```diff
@@ -70,6 +69,12 @@ def can_introspect_foreign_keys(self):
         "Confirm support for introspected foreign keys"
         return self._mysql_storage_engine != 'MyISAM'
 
+    @cached_property
+    def introspected_field_type(self):
+        introspected_field_type = super().introspected_field_type
+        introspected_field_type.update({'boolean': 'IntegerField'})
```

Comment:
You'll want to create a new `dict` instance here as you just happened to update `base.BaseDatabaseFeatures.introspected_field_type` in place.

Reply:
Thanks @charettes. I Forgot about it :man_facepalming:.
I created a new `dict` by calling `copy`